README.md

```diff
@@ -0,0 +1,239 @@
+# swo-cli command-line search client for SolarWinds Observability log management service
+
+Small standalone command line tool to retrieve and search recent app
+server metrics from [Solarwinds].
+
+### This is v1 of the swo-cli and it supports ONLY logs search.
+### This is v1 of the swo-cli and it DOES NOT support tailing.
+
+Supports optional Boolean search queries. Example:
+
+    $ swo-cli logs "(www OR db) (nginx OR pgsql) -accepted"
+
+Output is line-buffered so it can be fed into a pipe, like for grep.
+ANSI color codes are rendered in color on suitable terminals; see below for
+additional colorization options.
+
+## Quick Start
+
+Install [Go]
+
+    $ go install github.com/solarwinds/swo-cli@latest
+    $ echo "token: 123456789012345678901234567890ab" > ~/.swo-cli.yml
+    $ echo "api-url: https://api.na-01.cloud.solarwinds.com" >> ~/.swo-cli.yml
+    $ swo-cli
+
+Retrieve the full-access token from SolarWinds Observability.
+
+The API token can also be passed in the `SWOKEN`
```

Comment:
What is a SWOKEN?  Maybe we just call this SWO_API_TOKEN?

Reply:
renamed to `SWO_API_TOKEN`